<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Состояние регистра(m):</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="registerStateEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущая длина регистра:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="LengthRegister">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="decipherButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Зашифровать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Исходный текст:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Зашифрованный текст:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>291</width>
      <height>221</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="decipheredTextEdit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>301</width>
      <height>221</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Зашифрованный ключ:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="decipheredKeyEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>400</y>
      <width>471</width>
      <height>121</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>694</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="openAction">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
